components/Layout/Gnb/ChainSelect/Toggle.tsx

```diff
@@ -3,24 +3,25 @@ import clsx from 'clsx'
 
 import { getNetworkLabel } from 'utils/getNetworkLabel'
 import { isEthereum } from 'utils/isEthereum'
-import { useAuth, useChain, useConnect } from 'hooks'
+import { useAuth, useChain } from 'hooks'
 
 import { StyledGnbChainSelectToggle } from './styled'
 import CryptoIcon from 'components/CryptoIcon'
+import { useConnectModal } from '@rainbow-me/rainbowkit'
 
 type GnbChainSelectToggleProps = {
   toggle(e: MouseEvent<HTMLButtonElement>): void
 }
 
 function GnbChainSelectToggle({ toggle }: GnbChainSelectToggleProps) {
   const { isConnected } = useAuth()
-  const { openConnectModal } = useConnect()
+  const { openConnectModal } = useConnectModal()
   const { chainId } = useChain()
 
   const onClickToggle = useCallback(
     (e: MouseEvent<HTMLButtonElement>) => {
       if (isConnected) toggle(e)
-      else openConnectModal()
+      else openConnectModal?.()
     },
```

Comment:
```suggestion
      else if (openConnectModal){
        openConnectModal();
      }
```
`isConnected` in wagmi will be disable (in line 33) undefined modal case before click event can be occured, but just in case...
https://github.com/rainbow-me/rainbowkit/discussions/386#discussioncomment-7263219

Reply:
?.() 이런 식의 옵셔널 체이닝으로 작성하면 fn && fn()의 형태로 변형되어 제안주신 코드와 같은 역할을 할것으로 기대됩니다
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining